### PR TITLE
Revise welcome messages (shorter, less repetitive)

### PR DESCRIPTION
**Pull request by liewegas**

### CODE REVIEW

File: src/message_constants.ts [hunk 1]
```diff
@@ -40,7 +40,7 @@ export function CLARIFY_STATE(): string {
 export function WELCOME_VOTER(): string {
   switch (process.env.CLIENT_ORGANIZATION) {
     case 'VOTE_AMERICA':
-      return 'Welcome to VoteAmerica! We are excited to help you vote and can connect you to a trained volunteer to help answer your questions.\n\nReply HELPLINE to be connected to a volunteer. Reply STOP to unsubscribe from VoteAmerica texts. (Msg & data rates may apply).';
+      return "Welcome to VoteAmerica! We're excited to help you vote.\n\nReply HELPLINE to speak to a trained volunteer. Reply STOP to unsubscribe. (Msg&data rates may apply)";
```
Comment (tomerovadia) [inline src/message_constants.ts]: This is great. I'm a bit concerned that a voter might think "speak" implies a phone call. How about:

Welcome to VoteAmerica! We're excited to help you vote.\n\nReply HELPLINE to connect with a volunteer. Reply STOP to unsubscribe. (Msg & data rates may apply).

Comment (liewegas) [reply src/message_constants.ts]: updated. also modified the welcome + state message to fit in 160 chars

File: src/message_constants.ts [hunk 2]
```diff
@@ -40,7 +40,7 @@ export function CLARIFY_STATE(): string {
 export function WELCOME_VOTER(): string {
   switch (process.env.CLIENT_ORGANIZATION) {
     case 'VOTE_AMERICA':
-      return 'Welcome to VoteAmerica! We are excited to help you vote and can connect you to a trained volunteer to help answer your questions.\n\nReply HELPLINE to be connected to a volunteer. Reply STOP to unsubscribe from VoteAmerica texts. (Msg & data rates may apply).';
+      return "Welcome to VoteAmerica! We're excited to help you vote.\n\nReply HELPLINE to connect with a trained volunteer, or STOP to unsubscribe. (Msg&data rates may apply)";
```
Comment (tomerovadia) [inline src/message_constants.ts:43]: By my count this is 161 characters

File: src/message_constants.ts [hunk 3]
```diff
@@ -72,7 +72,7 @@ export function CLARIFY_HELPLINE_REQUEST(): string {
 
 // Only used by VOTE_AMERICA.
 export function WELCOME_AND_STATE_QUESTION(): string {
-  return 'Welcome to VoteAmerica! We are excited to help you vote and can connect you to a trained volunteer to help answer your questions.\n\nTo match you with the most knowledgeable volunteer, in which U.S. state are you looking to vote? (Msg & data rates may apply)\n\nReply STOP to unsubscribe from VoteAmerica texts.';
+  return 'Welcome to the VoteAmerica Helpline! We are excited to help. Msg&data rates may apply. Reply STOP to unsubscribe.\n\nIn which U.S. state are you looking to vote?';
```
Comment (tomerovadia) [inline src/message_constants.ts:75]: This also comes out to 161 characters by my count

Comment (tomerovadia) [inline src/message_constants.ts:75]: I think this is a great idea and this copy looks great to me (if we can squeeze into 160 chars). But can you run the final drafts by Snook and Brenna to make sure?

Comment (liewegas) [reply src/message_constants.ts:75]: I think you're off by the escape characters? Both are just under 160, and (re)confirmed from twilio logs that they're fitting in a single segment

Comment (tomerovadia) [reply src/message_constants.ts:75]: Ya you're right, thanks!

**Review by tomerovadia — CHANGES_REQUESTED** [review]
Can you change the PR name so that it includes the removal of the double-great? That way we can more easily see what changes happened with each PR / find changes.

**Review by tomerovadia — APPROVED** [review]

**Pull request merged**